Ryujinx.Horizon.Generators/CodeGenerator.cs

```diff
@@ -24,10 +24,10 @@ public void EnterScope(string header = null)
             IncreaseIndentation();
         }
 
-        public void LeaveScope()
+        public void LeaveScope(string suffix = "")
         {
             DecreaseIndentation();
-            AppendLine("}");
+            AppendLine("}" + suffix);
```

Comment:
Based on #3933, although it's probably not relevant here.
```suggestion
            AppendLine($"}}{suffix}");
```

Reply:
Otherwise $"" wouldn't work

Reply:
True! Sorry!